src/openepd/model/specs/generated/enums.py

```diff
@@ -0,0 +1,2426 @@
+     - Very high: Very high
+     - Extremely high: Extremely high
+
+    """
+
+    VERY_LOW = "Very low"
+    LOW = "Low"
+    MEDIUM = "Medium"
+    HIGH = "High"
+    VERY_HIGH = "Very high"
+    EXTREMELY_HIGH = "Extremely high"
+
+
+class CladdingFacingMaterial(StrEnum):
+    """
+    Cladding facing material.
+
+     - Steel: Steel
+     - Stone: Stone
+     - Aluminium: Aluminium
+     - Other: Other
+
+    """
+
+    STEEL = "Steel"
+    STONE = "Stone"
+    ALUMINIUM = "Aluminium"
+    OTHER = "Other"
+
+
+class WallBaseMaterial(StrEnum):
+    """
+    Wall base material.
+
+     - Rubber: Rubber
+     - Vinyl: Vinyl
+     - Other: Other
+
+    """
+
+    RUBBER = "Rubber"
+    VINYL = "Vinyl"
+    OTHER = "Other"
+
+
+class SprayFireproofingMaterialType(StrEnum):
+    """
+    Spray fireproofing material type.
+
+     - Gypsum-based: Gypsum-based
+     - Portland: Portland
+     - Gypsum and Portland: Gypsum and Portland
+
+    """
+
+    GYPSUM_BASED = "Gypsum-based"
+    PORTLAND_CEMENT_BASED = "Portland"
+    GYPSUM_AND_PORTLAND_CEMENT_BASED = "Gypsum and Portland"
+
+
+class MasonryCementAstmC91Type(StrEnum):
+    """
+    Masonry cement astm c91 type.
+
+     - Type N: Type N
+     - Type S: Type S
+     - Type O: Type O
+     - Type M: Type M
+
+    """
+
+    TYPE_N = "Type N"
+    TYPE_S = "Type S"
+    TYPE_O = "Type O"
+    TYPE_M = "Type M"
+
+
+class CementAstmType(StrEnum):
+    """
+    Cement astm type.
+
+     - C150 Type I: C150 Type I
+     - C150 Type I/II: C150 Type I/II
+     - C150 Type III: C150 Type III
+     - C150 Type IV: C150 Type IV
+     - C150 Type II/V: C150 Type II/V
+     - C595 PLC: C595 PLC
+     - C595 Blended: C595 Blended
+
+    """
+
+    C150_TYPE_I = "C150 Type I"
+    C150_TYPE_I_II = "C150 Type I/II"
+    C150_TYPE_III = "C150 Type III"
+    C150_TYPE_IV = "C150 Type IV"
+    C150_TYPE_II_V = "C150 Type II/V"
+    C595_PLC = "C595 PLC"
+    C595_BLENDED = "C595 Blended"
+
+
+class CmuBlockType(StrEnum):
+    """
+    Cmu block type.
+
+     - Gray: Gray
+     - Architectural: Architectural
+
+    """
+
+    GRAY = "Gray"
+    ARCHITECTURAL = "Architectural"
+
+
+class CarpetYarnType(StrEnum):
+    """
+    Carpet yarn type.
+
+     - Nylon 6,6: Nylon 6,6
+     - Nylon 6: Nylon 6
+     - Wool: Wool
+     - Recycled PET: Recycled PET
+     - Polyester (PET): Polyester (PET)
+     - Olefin: Olefin
+
+    """
+
+    NYLON_6_6 = "Nylon 6,6"
+    NYLON_6 = "Nylon 6"
+    WOOL = "Wool"
+    RECYCLED_PET = "Recycled PET"
+    POLYESTER_PET = "Polyester (PET)"
+    OLEFIN = "Olefin"
+
+
+class AluminiumAlloy(StrEnum):
+    """
+    AluminiumAlloy.
+
+     - 1xxx: 1xxx
+     - 2xxx: 2xxx
+     - 3xxx: 3xxx
+     - 4xxx: 4xxx
+     - 5xxx: 5xxx
+     - 6xxx: 6xxx
+     - 7xxx: 7xxx
+     - 8xxx: 8xxx
+     - 1xx.x: 1xx.x
+     - 2xx.x: 2xx.x
+     - 3xx.x: 3xx.x
+     - 4xx.x: 4xx.x
+     - 5xx.x: 5xx.x
+     - 7xx.x: 7xx.x
+     - 8xx.x: 8xx.x
+     - 9xx.x: 9xx.x
+
+    """
+
+    _1XXX = "1xxx"
```

Comment:
Can we avoid using an underscore at the beginning to prevent errors like "Access to a protected member _1XXX of a class"?

Reply:
I'll think how to better rename the enum members.